frontend/src/views/NewElementsPage/index.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div>
+    <DsfrNotice title="En cours de construction" desc="Plus de fonctionnalités à venir bientôt !" />
+    <div class="fr-container">
+      <DsfrBreadcrumb
+        class="mb-8"
+        :links="[
+          { to: { name: 'DashboardPage' }, text: 'Tableau de bord' },
+          { text: 'Demandes d\'ajout d\'ingrédient' },
+        ]"
+      />
+      <h1 class="fr-h4">Liste des demandes en attente d'ajout d'ingrédients</h1>
```

Comment:
Peut-être que "liste" et "en attente" n'apportent pas grande chose, de plus comme ça le titre serait cohérent avec celui du breadcrumb ?
```suggestion
      <h1 class="fr-h4">Demandes d'ajout d'ingrédients</h1>
```